<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComputeInterceptGradientDialog</class>
 <widget class="QDialog" name="ComputeInterceptGradientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Intercept and Gradient Grids</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ProjectViewer/icons.qrc">
    <normaloff>:/icons/images/logo_fs_64x64.png</normaloff>:/icons/images/logo_fs_64x64.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Parameters</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Dataset:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" colspan="2">
         <widget class="QComboBox" name="cbDataset"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Horizon:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3" colspan="2">
         <widget class="QComboBox" name="cbHorizon"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Reduction Method:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3" colspan="2">
         <widget class="QComboBox" name="cbMethod"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Number of Samples:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3" colspan="2">
         <widget class="QSpinBox" name="sbSamples">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>99</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>7</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QCheckBox" name="cbSuperGather">
          <property name="text">
           <string>Use Supergathers</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Inlines to Combine:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3" colspan="2">
         <widget class="QSpinBox" name="sbInlines">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="3">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Crosslines to Combine:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="3" colspan="2">
         <widget class="QSpinBox" name="sbCrosslines">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QCheckBox" name="cbRestrictOffset">
          <property name="text">
           <string>Restrict Offsets</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="8" column="0" colspan="4">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Minimum Offset:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="4">
         <widget class="QLineEdit" name="leMinOffset">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>-2000</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0" colspan="4">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Maximum Offset:</string>
          </property>
         </widget>
        </item>
        <item row="9" column="4">
         <widget class="QLineEdit" name="leMaxOffset">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>2000</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QCheckBox" name="cbRestrictAzimuth">
          <property name="text">
           <string>Restrict Azimuths</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>Minimum Azimuth:</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1" colspan="4">
         <widget class="QLineEdit" name="leMinAzimuth">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>Maximum Azimuth:</string>
          </property>
         </widget>
        </item>
        <item row="12" column="2" colspan="3">
         <widget class="QLineEdit" name="leMaxAzimuth">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>180</string>
          </property>
         </widget>
        </item>
        <item row="13" column="0" colspan="5">
         <widget class="QCheckBox" name="cbAVA">
          <property name="text">
           <string>Amplitude vs sin^2(theta)</string>
          </property>
         </widget>
        </item>
        <item row="14" column="0" colspan="4">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Average Reflector Depth:</string>
          </property>
         </widget>
        </item>
        <item row="14" column="4">
         <widget class="QLineEdit" name="leDepth">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Output</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Intercept:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="leIntercept"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Gradient:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="leGradient"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Quality:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="leQuality"/>
           </item>
           <item row="3" column="1">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="cbDisplayGradient">
          <property name="text">
           <string>Display Gradient Grid</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="cbDisplayIntercept">
          <property name="text">
           <string>Display Intercept Grid</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ProjectViewer/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ComputeInterceptGradientDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ComputeInterceptGradientDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbSuperGather</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbCrosslines</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbRestrictAzimuth</sender>
   <signal>toggled(bool)</signal>
   <receiver>leMinAzimuth</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbRestrictOffset</sender>
   <signal>toggled(bool)</signal>
   <receiver>leMaxOffset</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbSuperGather</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbInlines</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbRestrictAzimuth</sender>
   <signal>toggled(bool)</signal>
   <receiver>leMaxAzimuth</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbAVA</sender>
   <signal>toggled(bool)</signal>
   <receiver>leDepth</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbRestrictOffset</sender>
   <signal>toggled(bool)</signal>
   <receiver>leMinOffset</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>622</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
